examples/SLAM/README.md

```diff
@@ -0,0 +1,20 @@
+# SLAM (World tracking) camera demo
+![Preview](previews/SLAM-example.webp?raw=true "Dynamically generated mesh")
+Runs an AR session using WebXR Device AP where available with a fallback to 8th-wall implementation.
+
+To successfully run in 
+- make sure `API_TOKEN_XR8` is defined in the index.html file
```

Comment:
Note, this will move into `index.js` before 1.0.0 release... maybe as `window.API_TOKEN_XR8`?